src/Magento/FunctionalTestingFramework/Console/BaseGenerateCommand.php

```diff
@@ -28,6 +29,23 @@ protected function configure()
             'r',
             InputOption::VALUE_NONE,
             'remove previous generated suites and tests'
+        )->addOption(
+            "force",
+            'f',
+            InputOption::VALUE_NONE,
+            'force generation of tests regardless of Magento Instance Configuration'
```

Comment:
How about reword it a bit to include "run test", something similar like below for "allowSkip"?

Reply:
Good point, will do 👍